doc/index.rst

```diff
@@ -1,10 +1,44 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
-To get started using the Platform see :ref:`plutus_getting_started`.
+Intended Audience
+-------------------------------
 
+External Developers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This user guide is intended for external developers who are authoring distributed 
+applications ("DApps") that implement smart contracts on the Cardano blockchain. 
+
+.. note::
+   If you are a developer who wants to contribute to the Plutus Tools SDK project, 
+   please refer to documentation residing in the `Plutus Tools SDK repository <https://github.com/
+   input-output-hk/plutus-apps>`_.
+
+Plutus Tools SDK
+===============================
+
+The Plutus Tools SDK is a collection of off-chain infrastructure resources built 
+for external developers. The `Plutus Tools SDK repository <https://github.com/
+input-output-hk/plutus-apps>`_ supports the underlying resources that developers 
+need who are writing full applications using Plutus in Haskell, including off-chain 
+code. The term “off-chain code” refers to the part of a contract application’s code 
+which runs off the chain. 
+
+The `Plutus Tools SDK repository <https://github.com/input-output-hk/plutus-apps>`_ 
+contains packages such as: 
+
+* the `Plutus Application Backend (PAB) <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_, 
+  an off-chain application for managing the state of Plutus contract instances;
+
+* the `chain-index <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-chain-index>`_, 
+  a lightweight, customizable chain follower application and library for DApp 
+  developers who need to index and query the Cardano blockchain; 
+
+* a variety of other Plutus packages. 
```

Comment:
I think this says too little. Some things that I would like to know:
* What packages? Are they packages or Haskell modules?
* What are they used for?

I think just 1-2 examples would do, but without those this seems quite superfluous as it does not provide any new information.

Reply:
@raduom I agree with your comment. I would like to include more substantial information here, but I'm unclear about what is available today from Tools that is ready for developers to use. What else could we add here, and what else might we say about what is available? 

@michaelpj Excellent point. I would like to have a basic list of solid Tools offerings here in order to give developers a better sense of what Tools is. I'm ok with updating the list regularly if needed to keep it current. Alternatively, we could link from here to another page that has a more thorough list of available packages. I've talked with Konstantinos about such a list. Related tickets are PLT-444 and PLT-644. 

Are there 2 or 3 more examples we could add here? From the list of a couple of dozen folders on the plutus-apps repo, which ones, if any, do you think would be helpful to reference specifically in this intro? 

This page seems interesting. Would it make sense to mention some of what it mentions? 

https://github.com/input-output-hk/plutus-apps/blob/main/plutus-use-cases/ARCHITECTURE.adoc

Reply:
I was suggesting you remove that line. Currently we have: PAB and chain-index. Soon we will also have Marconi, and the PAB and chain-index versions based on Marconi.

Reply:
I think that listing the projects in plutus-tools with a brief explanation is a good idea, here.